packages/controls/src/controls/index.ts

```diff
@@ -16,6 +16,7 @@ export * from './text-area'
 export * from './text-input'
 export * from './font'
 export * from './typography'
+export * from './shape-v2'
```

Comment:
Can we nest this version under `shape/v2` the way we did it for rich text? This way our catalog of controls will stay more manageable as we add more and more versions.

Reply:
I can change this. 

I was kinda iffy on this decision because `RichText` has sub folders, but `StyleV2` doesn't.

Reply:
Yeah, we should do this to style, too.